test/autorest/complexgroup/inheritance_test.go

```diff
@@ -31,12 +31,12 @@ func TestInheritanceGetValid(t *testing.T) {
 		Breed: to.StringPtr("persian"),
 		Color: to.StringPtr("green"),
 		Hates: &[]complexgroup.Dog{
-			complexgroup.Dog{
```

Comment:
Why did this change? Again, just to know. Can we not declare the type here now?

Reply:
We can but it's redundant.

Reply:
To clarify, array of `type Dog struct`, each element can only be a `Dog` so no need to explicitly declare each element's type.

Reply:
Ok good to know, will keep in mind